assets/css/layouts/footer.css

```diff
@@ -77,3 +77,20 @@ ul.social-links {
   margin-top: -4px;
 }
 
+fieldset.color-scheme-toggle {
+  display: none;
+}
+
+.js-enabled fieldset.color-scheme-toggle {
+  display: inline-flex;
+  align-items: center;
+  gap: var(--spacing-md);
+  font-size: 1rem;
+  margin-top: var(--spacing-md);
+  border: none;
+  padding: 0;
+}
+
+fieldset.color-scheme-toggle span {
+  font-weight: bold;
+}
```

Comment:
If creating a separate nunjucks partial for the theme toggle then move this to its own css partial